package.json

```diff
@@ -130,22 +130,22 @@
     "babel-plugin-react-transform": "~1.1.1",
     "babel-runtime": "~5.8.29",
     "better-npm-run": "^0.0.4",
+    "bootstrap-loader": "^1.0.1",
     "bootstrap-sass": "^3.3.5",
-    "bootstrap-sass-loader": "^1.0.9",
     "chai": "^3.3.0",
+    "clean-webpack-plugin": "^0.1.4",
```

Comment:
Looks like npm alphabetized the devDependencies when I did `install --save-dev`. The only real changes are removing `bootstrap-sass-loader` and adding `bootstrap-loader` and `resolve-url-loader`.